src/Services/Flight/src/Flight/Aircrafts/Models/ValueObjects/Name.cs

```diff
@@ -0,0 +1,18 @@
+namespace Flight.Aircrafts.Models.ValueObjects;
+using System;
+
+public record Name : GenericValueObject<string>
```

Comment:
Please remove generic approach for this value object too.

Reply:
I'll work on implementing the recommended changes.

Reply:
done